### PR TITLE
[PLAY-846] Responsive spacing global props not falling back to defaults

### PR DESCRIPTION
**Pull request by nickamantia**
**What does this PR do?** A clear and concise description with your runway ticket url.

[PLAY-846](https://nitro.powerhrg.com/runway/backlog_items/PLAY-846)

**Screenshots:** Screenshots to visualize your addition/change

**How to test?** Steps to confirm the desired behavior:
4. See addition/change

#### Checklist:
- [ ] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [ ] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
- [ ] **TESTS** I have added test coverage to my code.

### CODE REVIEW

File: playbook-website/app/javascript/site_styles/docs/_kit_doc.scss [hunk 1]
```diff
@@ -43,6 +43,10 @@
     display: block;
   }
 
+  & > [class^=pb_card_body_kit] {
```
Comment (jasperfurniss) [inline playbook-website/app/javascript/site_styles/docs/_kit_doc.scss:46]: This piece is kinda odd. Trying to remember why this was done.
@nickamantia @carloslimasd 

Any ideas on why we needed to tweak the site scss?

Comment (nickamantia) [reply playbook-website/app/javascript/site_styles/docs/_kit_doc.scss:46]: @jasperfurniss Yeah that does seem odd. Ill take a look and see what happens when this is removed

Comment (nickamantia) [reply playbook-website/app/javascript/site_styles/docs/_kit_doc.scss:46]: @jasperfurniss @carloslimasd I cant seem to figure out what this does. Ive commented out the code and i don't see any changes. The only thing i can think of is i put this here when the the rails card was automatically adding the body sub component and then never removed this when we fixed that issue.

File: playbook-website/app/javascript/site_styles/docs/_kit_doc.scss [hunk 2]
```diff
@@ -43,6 +43,10 @@
...
+    padding: 0px;
+  }
```
Comment (jasperfurniss) [inline playbook-website/app/javascript/site_styles/docs/_kit_doc.scss:48]:
```suggestion
```

File: playbook/app/pb_kits/playbook/pb_card/card.html.erb [hunk 3]
```diff
@@ -4,9 +4,7 @@
     class: object.classname,
     aria: object.aria,
     dark: object.dark) do %>
-  <%= pb_rails("card/card_body", props: { padding: object.body_padding, flex_direction: object.body_flex_direction, flex_wrap: object.body_flex_wrap, justify_content: object.body_justify_content, justify_self: object.body_justify_self, align_items: object.body_align_items, align_content: object.body_align_content, align_self: object.body_align_self, flex: object.body_flex, flex_grow: object.body_flex_grow, flex_shrink: object.body_flex_shrink, order: object.body_order }) do %>
```
[the remark this answers is not in the document]
Comment (carloslimasd) [reply playbook/app/pb_kits/playbook/pb_card/card.html.erb:7]: We decided to stick with this structure change and Nitro test the impacts to make sure everything is still working.
Thank you @nidaqg

**Review by nidaqg — DISMISSED** [review]

**Review by nidaqg — APPROVED** [review]

**Pull request merged**